vendor/github.com/containers/image/v5/docker/docker_transport.go

```diff
@@ -43,30 +43,44 @@ func (t dockerTransport) ValidatePolicyConfigurationScope(scope string) error {
 
 // dockerReference is an ImageReference for Docker images.
 type dockerReference struct {
-	ref reference.Named // By construction we know that !reference.IsNameOnly(ref)
+	ref             reference.Named // By construction we know that !reference.IsNameOnly(ref)
+	isUnknownDigest bool
```

Comment:
`StringWithinTransport`, `*Policy*` (and tests) will need updating to handle these values.